ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-commands.sh

```diff
@@ -53,9 +53,13 @@ fi
 
 cd "\${REPO_DIR}"
 
+cat << VARS >> /root/config
```

Comment:
assuming declaration of ``DISCONNECTED`` in operator-gather-step-ref, I think this is not needed